prow_config.yaml

```diff
@@ -24,3 +24,25 @@ workflows:
     - test/*
     params:
       registry: "gcr.io/kubeflow-images-public"
+  - app_dir: kubeflow/katib/test/workflows
+    component: workflows-v1alpha2
+    name: e2e-v1alpha2
+    job_types:
+      - presubmit
+    include_dirs:
+    - pkg/*
+    - cmd/*
+    - test/*
+    params:
+      registry: "gcr.io/kubeflow-ci"
+  - app_dir: kubeflow/katib/test/workflows
+    component: workflows-v1alpha2
+    name: e2e-v1alpha2-release
+    job_types:
+      - postsubmit
```

Comment:
do we need postsubmit for v1alpha2 if the same image is built ?

Reply:
In future, I wonder if all the images containing both versions (at least, metricsCollectors may be dropped).  keep it as it is now, update if in need